vignettes/baggr_model_convergence.Rmd

```diff
@@ -0,0 +1,119 @@
+---
+title: "Understanding Model Convergence"
+author: "Brice Green"
+date: "`r Sys.Date()`"
+output:
+  rmarkdown::html_vignette:
+  toc: true
+vignette: >
+  %\VignetteIndexEntry{baggr_model_convergence}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+bibliography: baggr.bib
+---
+
+The `baggr` package makes running Bayesian meta-analysis models very easy, but that ease can also obscure some things that go on behind the scenes. Especially with low-data settings, say with only a few measurements, you may get cryptic error messages when you run your models, like
```

Comment:
Just _baggr_.

Please explain in the very first sentence what this vignette is about and who is it for: "This is a vignette intended for _baggr_ users who have no or little experience with Markov Chain Monte Carlo (MCMC) programs used for Bayesian inference. It is intended to help you understand common problems that may occur when running models. Especially in when there are few data you may see cryptic error messages when you run your models, like"

Reply:
Yep, that makes sense. I'll edit and re-push.